recipes/vtk/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "7.0.0" %}
+
+package:
+  name: vtk
+  version: {{ version }}
+
+source:
+  url: http://www.vtk.org/files/release/7.0/VTK-7.0.0.tar.gz
+  fn:  VTK-7.0.0.tar.gz
+  sha1: 7719fac36b36965eaf5076542166ba49bbe7afbb
+
+build:
+  number: 0
+  features:
+    - vc9                              # [win and py27]
+    - vc10                             # [win and py34]
+    - vc14                             # [win and py35]
+  skip: True                           # [osx]
+
+requirements:
+  build:
+    - gcc                              # [linux and (py27 or py34 or py35)]
```

Comment:
We built on a dev's machine, so hard to tell what the exact cause was. Given that these builds run on CI servers, we should be able to rely on the systems gcc version since the other packages do as well